src/ray/raylet/worker.h

```diff
@@ -100,6 +102,8 @@ class Worker {
   rpc::ClientCallManager &client_call_manager_;
   /// The rpc client to send tasks to this worker.
   std::unique_ptr<rpc::WorkerTaskClient> rpc_client_;
+  /// Whether the worker is persistent.
+  bool is_detached_;
```

Comment:
Let's call this `is_detached_actor_` and say in the documentation that this is only relevant for workers that are actors :)